test/e2e/play_kube_test.go

```diff
@@ -4093,7 +4093,7 @@ o: {{ .Options.o }}`})
 	It("persistentVolumeClaim with source", func() {
 		fileName := "data"
 		expectedFileContent := "Test"
-		tarFilePath := filepath.Join(os.TempDir(), "podmanVolumeSource.tgz")
+		tarFilePath := filepath.Join(podmanTest.TempDir, "podmanVolumeSource.tgz")
```

Comment:
Yuk. This was creating a file in $TMPDIR and never cleaning it up. Not just littering: this prevented running rootless after running as root.